src/saml.ts

```diff
@@ -267,6 +267,9 @@ class SAML {
 
     const samlExtensions = this.options.samlExtensions;
     if (samlExtensions != null) {
+      if (!(typeof samlExtensions == "object")) {
```

Comment:
Perhaps a bit of a nitpick, but I'm wondering why this construction which separates the `!` from the `==` instead of using `!=`.

Reply:
My bad :(, I am a little bit in a hurry. Adding fix :) thank you